<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnnemiesSet</class>
 <widget class="QDialog" name="EnnemiesSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="setColorPushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Couleur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="colorLineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Couleur </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Stats</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>56</y>
      <width>47</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>Vie : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Mana : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="vieSpinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>46</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>52</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>/ Vie max : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxLifeSpinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>46</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>76</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>/ Mana max : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="manaSpinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>76</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="manaMaxSpinBox">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>76</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>15</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Clochard de l'Espace</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodge : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dodgeSpinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>105</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>109</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Critique :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="critiqueSpinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>105</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>138</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Absorption :</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="absoDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>133</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.200000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>241</width>
     <height>192</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Incomming...</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Predefs : </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>301</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Attaque : </string>
   </property>
   <widget class="QCommandLinkButton" name="addAttaqueCommandLinkButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="attaqueNameLineEdit">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>15</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Pet de Mouche</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Dégâts :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="attaqueDegatsSpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>14</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>MP Cost :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="attaqueMpCostSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>45</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Heal :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="attaqueHealSpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Charge :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="attaqueChargeSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>67</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>68</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Persistence :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="attaquePersistenceSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>67</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dégât modifier :</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="attaqueDegatsModifierDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>97</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Armure modifier :</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="attaqueArmureModifierDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>96</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MPcost modif. :</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="attaqueMpCostModifierDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>126</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>128</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Abso modifier :</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="attaqueAbsoModifierDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>125</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>153</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodge hit modif. :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="attaqueDodgeHitModifierSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>153</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Crit hit modif. :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="attaqueCritHitModifierSpinBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>177</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Crit modifier :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="attaqueCritModifierSpinBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>174</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="fireSpinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>233</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Poison :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>184</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Feu :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="stunSpinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>252</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>208</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Glace :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>257</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Stun :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="poisonSpinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>228</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="freezeSpinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>204</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="fireLuckSpinBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="freezeLuckSpinBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>205</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="poisonLuckSpinBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>228</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="stunLuckSpinBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="attaquesList">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>401</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Attaques déjà existantes :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
